package.json

```diff
@@ -255,6 +252,9 @@
     "yargs-parser": "13.1.2",
     "yargs-unparser": "2.0.0"
   },
+  "dependencies": {
+    "copy-to-clipboard": "3.3.1"
+  },
```

Comment:
Should be this under `devDependencies`?

Reply:
no, copy-to-clipboard is used as app dependency